reactor-netty-core/src/main/java/reactor/netty/tcp/SslProvider.java

```diff
@@ -233,12 +244,16 @@ public static SslProvider defaultClientProvider() {
 		Builder sslContext(SslContext sslContext);
 
 		/**
-		 * The SslContextBuilder for building a new {@link SslContext}.
+		 * The SslContextBuilder for building a new {@link SslContext}. The default configuration is applied after
+		 * the custom configuration.
 		 *
 		 * @return {@literal this}
+		 * @deprecated as of 1.0.6. Prefer {@link #sslContext(ProtocolSslContextSpec)}, where the default
+		 * configuration is applied before any other custom configuration.
+		 * This method will be removed in version 1.2.0.
 		 */
+		@Deprecated
 		DefaultConfigurationSpec sslContext(SslContextBuilder sslCtxBuilder);
```

Comment:
Shouldn't `DefaultConfigurationSpec` itself be deprecated as well?

Reply:
fixed